heartbeat/awseip

```diff
@@ -116,10 +132,24 @@ END
 awseip_start() {
     awseip_monitor && return $OCF_SUCCESS
 
-    $AWSCLI ec2 associate-address  \
-        --instance-id ${INSTANCE_ID} \
-        --public-ip ${ELASTIC_IP}
-    RET=$?
+    ASSOCIATE_METHOD="ELASTIC_IP"
+    if [ -n "${ALLOCATION_ID}" ] && [ -n "${PRIVATE_IP_ADDRESS}" ]; then
+        ASSOCIATE_METHOD="ALLOCATION_ID"
+    fi
+
+    if [ "${ASSOCIATE_METHOD}" = "ALLOCATION_ID" ]; then
```

Comment:
Uhh... what a stupid dummy code, I was debugging with this block, I'll remove it later.